src/rpc/rawtransaction.cpp

```diff
@@ -387,6 +387,7 @@ static RPCHelpMan getislocks()
                             },
                         },
                     }},
+                    {RPCResult::Type::STR_HEX, "id", "Request ID"},
```

Comment:
let's add a functional test for this field to `interface_zmq_dash.py`?

The helper in python code for `get_request_id` already exist in `rpc_verifyislock.py`